setup.py

```diff
@@ -12,7 +12,7 @@
     url='https://github.com/mapbox/watchbot-progress-py',
     license='MIT',
     packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
-    install_requires=['boto3', 'redis'],
+    install_requires=['boto3', 'redis==2.10.6'],
```

Comment:
Can we leave the decision about the exact version up to the application, keeping the library requirement as loose as possible, like `redis~=2.0` to indicate compatibility with the 2.x releases.

Reply:
Glad you agree @vincentsarago :expressionless: